packages/api/test/e2e/app/app.e2e-spec.ts

```diff
@@ -1,14 +1,14 @@
 import {INestApplication} from '@nestjs/common';
 import {Test} from '@nestjs/testing';
 import * as request from 'supertest';
-import {AppModule} from "../../../src/app.module";
+import {ApplicationModule} from "../../../src/app.module";
```

Comment:
run the formatter on this file
```suggestion
import { ApplicationModule } from "../../../src/app.module";
```

Reply:
These are .spec files (for testing) which are ignored by ESLint. I can add them if you wan't but atm I think it doesn't matter that much :)